### PR TITLE
1 milestone - Mobile headline section

### PR DESCRIPTION
**Pull request by Nedramrez**
Set up a new repository
Set up linters
Add a .gitignore file
Add an index.html file
Add a styles.css file
Add Assets folder with images
Add a README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+      <div class="toolbar">
+        <nav>
+          <div class="normal-button">
+            <img class="burger" src="assets/burger.svg" alt="Menu icon.">
+          </div>
+        </nav>
+        <a href="#home">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
```
Comment (zieeco) [inline index.html:28]: - Please, kindly ensure your logo is wrapped inside the nav bar to meet the project requirement.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+        Glad to see <br>
+        you!</h1>
+      </div>      
+      <div class="social-media-icons">
+        <ul class="social-media-icons-list">
+          <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="20" height="20"></a></li>
```
Comment (zieeco) [inline index.html]: - Please, kindly ensure this links to your social media platform to help recruiters see your other professional social media presence. You add the ones you have and leave the ones you don't have yet as you'd update them later in the future.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+-  GitHub: [@Nedramrez](https://github.com/Nedramrez)
+-  Twitter: [@zermarden](https://twitter.com/zermarden)
+-  LinkedIn: [LinkedIn](https://linkedin.com/in/zermarden)
 
 ## 🤝 Contributing
 
-I want to say thank you to my learning partners, morning session and standup teams!
+I want to say thank you to all my learning partners, morning session and standup teams!
 
 ## Show your support
```
Comment (zieeco) [inline README.md:27]: *LICENSE*

- Please, kindly add licensed content as your link is broken to make your readme more professional

File: README.md [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
-- Twitter: [@zermarden](https://twitter.com/zermarden)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/zermarden)
+-  GitHub: [@Nedramrez](https://github.com/Nedramrez)
+-  Twitter: [@zermarden](https://twitter.com/zermarden)
+-  LinkedIn: [LinkedIn](https://linkedin.com/in/zermarden)
 
 ## 🤝 Contributing
 
-I want to say thank you to my learning partners, morning session and standup teams!
+I want to say thank you to all my learning partners, morning session and standup teams!
```
Comment (zieeco) [inline README.md:22]: - This should also contain a link to the issues page of your portfolio repo on Github. please update this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Nedramrez,


Grea job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — APPROVED** [review]
Hi @Nedramrez  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors 🚀 
- Descriptive commits , keep rocking 🥇 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**